pandas/_libs/src/datetime/np_datetime.c

```diff
@@ -564,18 +564,15 @@ void pandas_datetime_to_datetimestruct(npy_datetime val, PANDAS_DATETIMEUNIT fr,
 
 void pandas_timedelta_to_timedeltastruct(npy_timedelta val,
                                          PANDAS_DATETIMEUNIT fr,
-                                          pandas_timedeltastruct *result) {
+                                         pandas_timedeltastruct *result) {
   pandas_datetime_metadata meta;
 
   meta.base = fr;
-  meta.num - 1;
+  meta.num = 1;
```

Comment:
Thanks for fixing this typo. Do we even need the assignment at all though? Worth considering removal?

Reply:
I'm advocating moving the timedelta_struct stuff out of src/ entirely.  The src/ directory is a PITA and we can put it in cython directly without taking a performance hit.